hazelcast/src/main/java/com/hazelcast/query/impl/Indexes.java

```diff
@@ -241,6 +249,21 @@ public InternalIndex[] getIndexes() {
         return indexes;
     }
 
+    public List<StepAwareStorage> getStepAwareStorages() {
```

Comment:
moving this method to `EnterpriseMapContainer` could be more appropriate.

Reply:
Fixed in https://github.com/hazelcast/hazelcast/pull/25208/commits/ebd4bb4355716835d56b2901079be093e91b47db